async_fuse/src/protocol.rs

```diff
@@ -149,7 +149,9 @@ pub const FATTR_FLAGS: u32 = 1 << 31;
 // FOPEN_CACHE_DIR: allow caching this directory
 // FOPEN_STREAM: the file is stream-like (no file position at all)
 //
+#[allow(dead_code)]
```

Comment:
for the const variables, maybe it's better to use a mod to group them, and add allow dead_code to the mod.

Reply:
Done in <https://github.com/datenlord/datenlord/pull/68/commits/f1168fc807b1e96c2d0db1b04c8122dece7b8e44>